ui/src/app/job-details/panels/panels.component.html

```diff
@@ -28,6 +28,9 @@
       Ended: {{ job.end | jmShortDateTime }}
       ({{ job.start | jmDuration: job.end }})
     </p>
+    <p *ngIf="job.extensions && job.extensions.timingUrl">Graph View:<a href="{{ job.extensions.timingUrl }}" target="_blank">
+      <div class="title-link">Timing Diagram</div>
```

Comment:
I wouldn't put this on two lines because it can get a little scrunched for room:
<p>
![screen shot 2018-08-01 at 5 46 13 pm](https://user-images.githubusercontent.com/1713505/43550870-fe3a6886-95b2-11e8-8b9f-0a6f97870ed6.png)
</p>

Reply:
1. Fun fact - I actually intended it to be on a single line but wasn't sure what incantation of HTML tags would achieve that
2. I see you have extra `Started` and `Ended` times which I'm missing. I wonder whether I should look into why those don't exist for me (since on *my* Status panel, there's plenty of room!)